README.md

```diff
@@ -40,21 +40,22 @@
   - [`goal_src/`](#goal_src)
   - [`game` runtime](#game-runtime)
 
-## READ FIRST
+## Please read first
 
-The github repositories are for development of the project and tracking active issues.  Most of the information you will find here pertains to setting up the project for development purposes and is not relevant to a general user.
+Our repositories on GitHub are primarily for development of the project and tracking active issues. Most of the information you will find here pertains to setting up the project for development purposes and is not relevant to the end-user. The Releases page will provide you with a pre-compiled set of tools you need to get this project running on your machine. 
```

Comment:
let's avoid recommending people to use the releases page, long term this will not be the case so it's just something we'll have to update in a month or so.